pkgs/applications/video/epgstation/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, makeWrapper, bash, nodejs }:
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+  workaround-opencollective-buildfailures = stdenv.mkDerivation {
+    # FIXME: This should be removed when a complete fix is available
+    # https://github.com/svanderburg/node2nix/issues/145
+    name = "workaround-opencollective-buildfailures";
+    phases = [ "installPhase" ];
+    installPhase = ''
+      mkdir -p $out/bin
+      touch $out/bin/opencollective-postinstall
+      chmod +x $out/bin/opencollective-postinstall
+    '';
+  };
+in
+nodePackages."epgstation-git+https://github.com/l3tnun/EPGStation.git#v1.7.4".override (drv: {
+  buildInputs = [ bash ];
+  nativeBuildInputs = [
+    workaround-opencollective-buildfailures
+    makeWrapper
+    nodePackages.node-pre-gyp
+  ];
+
+  preRebuild = ''
+    patch -p1 <<<"${builtins.readFile ./epgstation.patch}"
+  '';
+
+  postInstall = let
+    runtimeDeps = [ nodejs bash ];
+  in
+  ''
+    mkdir -p $out/{bin,libexec,share/doc/epgstation}
+
+    pushd $out/lib/node_modules/EPGStation
+
+    npm run build
+
+    mv config/{enc.sh,enc.js} $out/libexec
+    mv LICENSE Readme.md $out/share/doc/epgstation
+    mv doc/* $out/share/doc/epgstation
+    rm -r config data doc logs recorded thumbnail
+
+    ln -sfT /etc/epgstation config
```

Comment:
Does this work? Looks like it tries to link to the global `/etc`.

Reply:
Yes, I have confirmed this works. The destination file path doesn't have to exist at the time of build for symlinks to be created.